content/knowledge_base/tips/setting_up_sftp.md

```diff
@@ -47,46 +49,54 @@ Match User sftpupload
     X11Forwarding no
 ```
 
-5. Create keys for the `sftpupload` user
+and then restart the `sshd` service:
 
+```sh
+$ sudo systemctl restarts sshd
 ```
-sudo -u sftpuser bash
+
+5. Create keys for the `sftpupload` user
+
+```sh
+sudo -u sftpupload bash
 $ ssh-keygen
 Generating public/private rsa key pair.
 Enter file in which to save the key (/home/sftpuser/.ssh/id_rsa)
 
-# Make sure the permissions are correct for the directory
-chmod 600 /home/sftpuser/.ssh/
-```
-
-In the offline collector configuration you should use this private key
-(`/home/sftpuser/.ssh/id_rsa`) of the form:
+# Authorize the user's public key for access
+$ cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
 
+# Make sure that secure permissions are applied for the directory
+$ chmod -v 600 /home/sftpupload/.ssh/
 ```
------BEGIN OPENSSH PRIVATE KEY-----
-.....
------END OPENSSH PRIVATE KEY-----
-```
-
 
-6. Verify you can connect to the server, list files and upload files
+6. Verify you can connect to the server and upload files. Listing files will be
+   denied.
 
-```
+```sh
 $ sftp localhost
-The authenticity of host 'localhost (127.0.0.1)' can't be established.
-ED25519 key fingerprint is SHA256:nJ9IXQjeXVXURD0bVCcylr4+5/Da0jnJEdrLqgZYBko.
-This key is not known by any other names
-Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
-Warning: Permanently added 'localhost' (ED25519) to the list of known hosts.
-Connected to localhost.
-sftp> ls -l files
-remote readdir("/files/"): Permission denied
+
 sftp> put /etc/passwd /files/passwd.txt
 Uploading /etc/passwd to /files/passwd.txt
+
+sftp> ls -l files
+remote readdir("/files/"): Permission denied
```

Comment:
can we also please add the test for connection with plain ssh to prove that you cant get a shell?